Formula/mariadb@10.9.rb

```diff
@@ -36,7 +36,8 @@ class MariadbAT109 < Formula
   keg_only :versioned_formula
 
   # See: https://mariadb.com/kb/en/changes-improvements-in-mariadb-109/
-  deprecate! date: "2023-08-01", because: :unsupported
+  # and https://mariadb.org/about/#maintenance-policy
+  deprecate! date: "2023-08-22", because: :unsupported
```

Comment:
Updated to use official EOL date from https://mariadb.org/about/#maintenance-policy

Release series | Stable (GA) Date | End-of-life
--- | --- | ---
[10.7](https://mariadb.com/kb/en/changes-improvements-in-mariadb-107/) | 9 Feb 2022 | 9 Feb 2023
[10.8](https://mariadb.com/kb/en/changes-improvements-in-mariadb-108/) | 20 May 2022 | 20 May 2023
**[10.9](https://mariadb.com/kb/en/changes-improvements-in-mariadb-109/)** | 22 Aug 2022 | **22 Aug 2023**
[10.10](https://mariadb.com/kb/en/changes-improvements-in-mariadb-1010/) | 17 Nov 2022 | 17 Nov 2023
[11.0](https://mariadb.com/kb/en/changes-and-improvements-in-mariadb-11-0/) | In development | 1 Year after GA